config/swagger-lume.php

```diff
@@ -201,4 +201,11 @@
     'constants' => [
         'SWAGGER_LUME_CONST_HOST' => env('SWAGGER_LUME_CONST_HOST', 'http://my-default-host.com'),
     ],
+    
+    /*
+    |--------------------------------------------------------------------------
+    | Force assets to be loaded over HTTPS (Solves mixed content errors when application is behind a load balancer.
+    |--------------------------------------------------------------------------
+     */
+    'force_https' => false,
```

Comment:
Can we have `env` var for this:
```suggestion
    'force_https' => env('SWAGGER_LUME_FORCE_HTTPS', false),
```

Reply:
Resolved in latest commit.